src/sentry/api/endpoints/organization_fork.py

```diff
@@ -34,6 +34,12 @@
 ERR_CANNOT_FORK_INTO_SAME_REGION = Template(
     "The organization already lives in region `$region`, so it cannot be forked into that region."
 )
+ERR_CANNOT_FORK_FROM_REGION = Template(
+    "Forking an organization from region `$region` is forbidden."
+)
+
+# For legal reasons, there are certain regions from which forking is disallowed.
+CANNOT_FORK_FROM_REGION = {"de"}
```

Comment:
So once you're in de you can never relocate somewhere else?

Reply:
Yes. IANAL, but my understanding is that's what we want?

Reply:
Ok. That could make for some tricky support issues, but we'll figure that out later.